modules/lib/lib-schema/src/main/resources/lib/xp/examples/schema/createMixin.js

```diff
@@ -0,0 +1,55 @@
+var schemaLib = require('/lib/xp/schema');
+var assert = require('/lib/xp/testing');
+
+/* global log*/
+
+let resource = `<?xml version="1.0" encoding="UTF-8"?>
+       <mixin xmlns="urn:enonic:xp:model:1.0">
+          <display-name>Virtual Mixin</display-name>
+          <description>Mixin description</description>
+          <form>
+            <input type="TextLine" name="text2">
+              <label>Text 2</label>
+            </input>
+        
+            <mixin name="myapplication:inline"/>
+          </form>
+        </mixin>`
+
+// BEGIN
+// Create virtual mixin.
+var result = schemaLib.createSchema({
+    name: 'myapp:mytype',
+    type: 'MIXIN',
+    resource: resource
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
    resource
```